DOCSP-44292: improve i&h (#51)
mongodb/docs-kotlin-sync: source/issues-and-help.txt

You can also open a case in Jira, our issue management tool, to identify
bugs or propose improvements. The following steps describe how to create
a Jira issue:

1. Visit the `MongoDB Jira issue tracker <https://jira.mongodb.org/>`__ and click the 
   `signup link. <https://account.mongodb.com/account/register>`__
   Create an account, and then log in to Jira.
#. Navigate to the `JAVA Jira project. <https://jira.mongodb.org/browse/JAVA>`__
#. Click :guilabel:`Create` to create a ticket. Please provide as much
   information as possible about the issue or request in the ticket.

.. note::

   Bug reports in the JAVA Jira project are publicly viewable.

If you’ve identified a security vulnerability in any official MongoDB
product, please report it according to the instructions found in the
:manual:`Create a Vulnerability Report page. </tutorial/create-a-vulnerability-report>`

Pull Requests
-------------

We are happy to accept contributions to help improve the driver. We will guide
user contributions to ensure they meet the standards of the codebase. Please
ensure that any pull requests include documentation, tests, and pass the
**gradle** checks.

> To get started check out the source and work on a branch:
> 
> .. code-block:: bash
> 
>    $ git clone https://github.com/mongodb/mongo-java-driver.git
>    $ cd mongo-java-driver
>    $ git checkout -b myNewFeature

Reviewer: S: I think you can reword this code intro slightly; also, you can remove the $ from the code example

```suggestion
To get started, clone the source repository and work on a branch by running the following commands:

.. code-block:: bash

   git clone https://github.com/mongodb/mongo-java-driver.git
   cd mongo-java-driver
   git checkout -b myNewFeature
```

> Finally, ensure that the code passes gradle checks.

Reviewer: S:
```suggestion
Finally, ensure that the code passes gradle checks by running the following command:
```

.. code-block:: bash

>    $ ./gradlew check

Reviewer: S:
```suggestion
   ./gradlew check
```
